ui/app/pages/send/send-content/send-content.component.js

```diff
@@ -77,13 +79,13 @@ export default class SendContent extends Component {
     );
   }
 
-  renderError() {
+  renderError(gasError = false) {
     const { t } = this.context;
     const { error } = this.props;
 
     return (
       <Dialog type="error" className="send__error-dialog">
-        {t(error)}
+        {t(gasError ? 'gasPriceExcessive' : error)}
```

Comment:
Nit: We're already breaking the "pass string literals directly to `t(..)`" rule here, but nonetheless it would be better to follow this guideline where possible to make our work easier in the future, when we migrate more parts of the UI to use that strict checker.

```suggestion
        {gasError ? t('gasPriceExcessive') : t(error)}
```

Reply:
done